server/pulp/plugins/loader/api.py

```diff
@@ -484,14 +485,35 @@ def _load_type_descriptors(path):
     return descriptors
 
 
-def _load_type_definitions(descriptors, drop_indices=False):
+def _load_type_definitions(definitions, drop_indices=False):
```

Comment:
I added comments and then decided to remove this one-line function ;) so thank you for your question)